packages/splunk/data_stream/alert/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,1885 @@
+      value: '{{{splunk.alert.src_user_last}}}'
+      allow_duplicates: false
+      if: ctx.splunk?.alert?.src_user_last instanceof String
+  - append:
+      field: related.user
+      tag: append_alert_user_first_into_related_user
+      value: '{{{splunk.alert.user_first}}}'
+      allow_duplicates: false
+      if: ctx.splunk?.alert?.user_first instanceof String
+  - append:
+      field: related.user
+      tag: append_alert_user_managedby_into_related_user
+      value: '{{{splunk.alert.user_managedBy}}}'
+      allow_duplicates: false
+      if: ctx.splunk?.alert?.user_managedBy instanceof String
+  - append:
+      field: related.user
+      tag: append_alert_user_nick_into_related_user
+      value: '{{{splunk.alert.user_nick}}}'
+      allow_duplicates: false
+      if: ctx.splunk?.alert?.user_nick instanceof String
+  - append:
+      field: tags
+      value: preserve_original_event
+      allow_duplicates: false
+      if: ctx.error?.message != null
+  - remove:
+      field:
+        - splunk.alert.annotations.mitre_attack.mitre_tactic
+        - splunk.alert.annotations.mitre_attack.mitre_tactic_id
+        - splunk.alert.annotations.mitre_attack.mitre_technique
+        - splunk.alert.annotations.mitre_attack.mitre_technique_id
+        - splunk.alert.city
+        - splunk.alert.creation_timestamp
+        - splunk.alert.dest
+        - splunk.alert.dest_ip
+        - splunk.alert.dest_port
+        - splunk.alert.device
+        - splunk.alert.devid
+        - splunk.alert.domain
+        - splunk.alert.dst_location
+        - splunk.alert.dstip
+        - splunk.alert.dstport
+        - splunk.alert.email
+        - splunk.alert.file_hash
+        - splunk.alert.file_name
+        - splunk.alert.file_path
+        - splunk.alert.file_size
+        - splunk.alert.file_type
+        - splunk.alert.host
+        - splunk.alert.hostname
+        - splunk.alert.id
+        - splunk.alert.ip
+        - splunk.alert.md5
+        - splunk.alert.os
+        - splunk.alert.os_family
+        - splunk.alert.os_version
+        - splunk.alert.protocol
+        - splunk.alert._raw
+        - splunk.alert.reason
+        - splunk.alert.sha256
+        - splunk.alert.src
+        - splunk.alert.src_ip
+        - splunk.alert.src_location
+        - splunk.alert.src_port
+        - splunk.alert.src_user_email
+        - splunk.alert.src_user_identity_id
+        - splunk.alert.src_user_nick
+        - splunk.alert.srcip
+        - splunk.alert._time
+        - splunk.alert.url
+        - splunk.alert.user
+        - splunk.alert.user_email
+        - splunk.alert.user_identity_id
+        - splunk.alert.userip
+        - splunk.alert.username
+      tag: remove_custom_duplicate_fields
+      ignore_missing: true
+      if: ctx.tags == null || !(ctx.tags.contains('preserve_duplicate_custom_fields'))
+  - remove:
+      field:
+        - _temp
+        - tmp_host
+        - temp_ip
+        - _temp_dest
```

```suggestion
  - remove:
      field:
        - _temp
```
Try nesting all temporary fields inside `_temp`.